libs/sdk-core/src/invoice.rs

```diff
@@ -147,10 +147,8 @@ pub fn add_lsp_routing_hints(
 
 /// Parse a BOLT11 payment request and return a structure contains the parsed fields.
 pub fn parse_invoice(bolt11: &str) -> Result<LNInvoice> {
-    let signed = bolt11
-        .strip_prefix("lightning:")
-        .unwrap_or(bolt11)
-        .parse::<SignedRawInvoice>()?;
+    let bolt11 = bolt11.strip_prefix("lightning:").unwrap_or(bolt11);
```

Comment:
Shouldn't we also strip LIGHTNING (ignore case)?

Reply:
Changed to a case insensitive replace